README.md

```diff
@@ -45,6 +45,67 @@ you can enable/disable for the command-line app, use the `--help` option:
 addons-linter --help
 ```
 
+### Import Linter API into another NodeJS application
+
+```js
+import {createInstance as createLinter} from 'addons-linter';
```

Comment:
If `createInstance` isn't a good name we should change it inside the linter rather than have our docs advocate importing it as a different name. I think this looks odd 🤷‍♀️